.gitignore

```diff
@@ -51,7 +51,7 @@ install/**
 External/install/**
 com.unity.formats.alembic/.npmignore
 com.unity.formats.alembic/package.json
-com.unity.formats.alembic/Runtime/Plugins
+#com.unity.formats.alembic/Runtime/Plugins
```

Comment:
Just remove the line

Reply:
This is temporary. As soon as I have a valid linux CI, this will be put back.